setup.md

```diff
@@ -39,53 +39,48 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
 
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
-- [Git](https://git-scm.com/downloads)
-- [Go](https://go.dev/doc/install) (v1.20+)
-- [GoReleaser](https://goreleaser.com/)
-- [Make](https://www.gnu.org/software/make/)
 - [jq](https://stedolan.github.io/jq/download/)
 
-## Step 1: Clone GUAC
+## Step 1: Download GUAC
 
-1. Clone GUAC to a local directory:
+1. Download the GUAC CLI `guacone` binary for your machine's OS and architecture
+   from the
+   [latest GUAC release](https://github.com/guacsec/guac/releases/latest). For
+   example Linux x86_64 is
+   [`guacone-linux-amd64'](https://github.com/guacsec/guac/releases/latest/download/guacone-linux-amd64).
 
-   ```bash
-   git clone https://github.com/guacsec/guac.git
-   ```
+1. Rename the binary to `guacone`, mark it executable if necessary, and add it
```

Comment:
not sure if we need to give this much detail but should we add how to mark it executable and add it to shell's path for inexperienced users?

Reply:
To do it thoroughly, we would need a matrix of instructions for OS and even Linux distribution, (ex: do you have ~/.profile or ~/.bash_profile ...). There is probably somewhere to link out to where someone has done this well. I'll do some searching.

About the separate doc. All the demos just need this setup.md done with compose up and guacone in the path. Users shouldn't need to download all the binaries if they don't need to for just the demos. I think a "run guac from scratch" with a tutorial on what each binary does would be a good doc to have, but have it be a separate thing, not a prereq.